### PR TITLE
Updates for libsql URLs

### PR DESCRIPTION
**Pull request by CodingDoug**

### CODE REVIEW

File: reference/libsql-urls.md [hunk 1]
```diff
@@ -41,22 +46,56 @@ instances over time depending on how Fly.io observes network latency.
...
+Turso databases support only the secure `https` and `wss` schemes. `ws` and
+`http` are typically only used when developing against a sqld instance running
+locally that can't be configured with an SSL certificate.
+
+:::
 
-## HTTP URLs
+The underlying data sent using websockets or HTTP is expressed as JSON.
+Documentation for this encoding and the structure of its messages is
+forthcoming.
```
Comment (honzasp) [inline reference/libsql-urls.md:90]: Nit: I would describe Hrana as "protocol", not an "encoding".

**Review by honzasp — APPROVED** [review]
Looks good :)

File: reference/libsql-urls.md [hunk 2]
```diff
@@ -41,22 +46,56 @@ instances over time depending on how Fly.io observes network latency.
...
+`turso db show $DBNAME --instance-urls`. Database instance URLs have the
+following format:
+
+```
+libsql://[RANDOM-CHARS]-[DB-NAME]-[ORG-NAME].turso.io
+```
+
+## Network protocol
+
+libSQL client libraries are free to choose what they consider to be the best (or
```
Comment (glommer) [inline reference/libsql-urls.md:58]: client libraries are free to choose what they consider the best -- _when using libsql URLs_, no ?

Comment (CodingDoug) [reply reference/libsql-urls.md:58]: Clarified.

**Pull request merged**